samples/nrf9160/http_update/modem_delta_update/Kconfig

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "HTTP modem delta update sample"
+
+config DOWNLOAD_FILE_BASE_TO_FOTA_TEST
+	string "File to update modem from base to test firmware"
+
+config DOWNLOAD_FILE_FOTA_TEST_TO_BASE
+	string "File to update modem from test firmware base"
+
+config DOWNLOAD_HOST
+	string "Host part of URL to download files."
+
+config SUPPORTED_BASE_VERSION
+	string "Supported base version"
```

Comment:
Would "Current base version" be correct? Or just "Current version"? Since this is a sample, I feel that a sentence or two of more explanation on some of these configs would be helpful.

Reply:
We might be in the situation where the user has modem version N installed, but the deltas configured by default is made for version N + 1. In this case we will print the "not supported version, install version N + 1 ... " etc. 

Hence it makes sense to me that this is called "Supported" since we only used it to ensure that the configured deltas supports this version.

Does that make sense? 

I will expand the docs on this.

Reply:
> Does that make sense?

Haha no, not really. Eagerly awaiting more docs :D.